drivers/frequency/admv1013/admv1013.h

```diff
@@ -99,9 +99,6 @@
 #define ADMV1013_SPI_READ_CMD			NO_OS_BIT(7)
 #define ADMV1013_SPI_WRITE_CMD			(0 << 7)
 
-#define MIXER_GATE_0_to_1_8_V(x)		((2389 * x/ 1000000 + 8100) / 100)
```

Comment:
i'd add a prefix for these also. so we keep them in the header file.

Reply:
are they useful in the header ? after all they're used by some update() function so the user doesn't need these macros

Reply:
Not really, just to keep it consistent with other drivers where we hold all macros in the header file.